docs/Quality.md

```diff
@@ -94,12 +94,58 @@ Good details in a bug report include:
   * Trivial (eg. a copy error)
 
 ### Bug Triage
-Bugs should be treated similarly as issues and scheduled as a part of Sprint grooming and planning. Showstoppers and certain Major issues will likely require immediate attention. All other severities should be mixed in with stories and scheduled appropriately.
+Bugs should be treated similarly to issues and scheduled as a part of Sprint grooming and planning. Showstoppers and certain Major issues will likely require immediate attention. All other severities should be mixed in with stories and scheduled appropriately.
 
-## Continuous Integration
+### Agile Manual QA Overflow
+Developing automated integration or UI tests are time consuming. It's almost impossible to maintain automation tests while a feature is being developed. And often, having enough time to create UI and integration tests while developing is a luxury we don't get. Even though your development process might include TDD, unit tests won't catch the bugs that occur due to integration. So, to continue to maintain high quality, manual QA becomes necessary until the feature settles down.
```

Comment:
"It's almost impossible to maintain automation tests while a feature is being developed." I disagree with this sentence.

Reply:
How about "hard to maintain"?

Reply:
That works.